client/src/Components/BoxBooks/BoxBooks.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from 'react';
```

Comment:
Czy to jest używane?

Reply:
racja - ostatecznie nie jest - wyrzuciłem